apisix/plugins/limit-count.lua

```diff
@@ -40,11 +40,11 @@ end
 
 function _M.workflow_handler()
     workflow.register(plugin_name,
-    function (conf, ctx)
-        return limit_count.rate_limit(conf, ctx, plugin_name, 1)
-    end,
     function (conf)
         return limit_count.check_schema(conf)
+    end,
+    function (conf, ctx)
+        return limit_count.rate_limit(conf, ctx, plugin_name, 1)
```

Comment:
Are there any other plugins that need to be modified?

Reply:
No these are the only plugins